internal/errors/errors.go

```diff
@@ -0,0 +1,37 @@
+package errors
+
+import "fmt"
+
+// Все возможные ошибки при работе с данными.
+var (
+	ErrInvalidDataFormat    = fmt.Errorf("invalid data format")
```

Comment:
Интересный способ, обычно делают так
```suggestion
	ErrInvalidDataFormat    = errors.New("invalid data format")
```